modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/life.dm

```diff
@@ -128,6 +132,18 @@
 		AddBloodVolume(((bruteheal * -0.5) + (fireheal * -1)) * costMult * mult) // Costs blood to heal
 		return TRUE
 
+/datum/antagonist/bloodsucker/proc/OverfeedHealing(drunk)
+	var/mob/living/carbon/user = owner.current
+	if(blood_over_cap > 0) //Checks if you are over your blood cap
+		var/overbruteheal = user.getBruteLoss_nonProsthetic()
+		var/overfireheal = user.getFireLoss_nonProsthetic()
+		var/heal_amount = drunk / 3
+		if(overbruteheal + overfireheal > 0)
+			user.adjustBruteLoss(-heal_amount, forced=TRUE) // Heal BRUTE / BURN in random portions throughout the body; prioritising BRUTE.
+			heal_amount = (heal_amount - overbruteheal) / 1.5 // Removes the amount of BRUTE we've already healed from the heal amount and then reduces it further (BURN should be harder to heal)
+			if(heal_amount > 0)
+				user.adjustFireLoss(-heal_amount, forced=TRUE)
```

Comment:
Make it return instead of healing brute and then checking if it needs to heal burns if you want it to prioritize brute first i'd reccomend, you don't really need to heal both in the same tick

Reply:
Unsure what you mean here but you said it's fine if a bit weirdly done to me over discord when i updated it so i'm resolving.